docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst

```diff
@@ -1,31 +1,126 @@
 .. _intervention_crm_mgmt_polypill:
 
 
-Vivarium Modeling Strategy - Polypill Intervention 
---------------------------------------------------
+=====================
+Polypill Intervention
+=====================
 
-.. todo::
+.. contents::
+   :local:
+   :depth: 1
 
-  Add an overview of the Vivarium modeling section.
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
 
-.. todo::
-  Address comment: I expect this to reference the hypertension treatment doc and describe how some ramp positions can be achieved by a polypill; it should say what the effect of polypill is on adherence, and who gets polypill (perhaps with reference to the scenarios.rst description of what the coverage of polypill is in your alternative scenarios.)
+  * - Abbreviation
+    - Definition
+    - Note
+  * - SBP
+    - Systolic Blood Pressure
+    - 
+  * - LDL-C
+    - Low-Density Lipoprotein Cholesterol
+    - 
+  * - CVD
+    - Cardiovascular Disease 
+    - 
 
-Decision tree and algorithm for polypill intervention
 
-.. toctree::
-   :maxdepth: 1
+Intervention Overview
+---------------------
 
-   visit
-   tx
+The polypill intervention combines multiple medications that would have been taken separately 
+into one prescription and a single pill. For this research, the polypill with include: 
+atorvastatin (10 mg), amlodipine (2.5 mg), losartan (25 mg), and hydrochlorothiazide (12.5 mg). 
+All of these are hypertension medications, although there are trials that combine hypertension 
+medications and statins. 
 
+Today, few patients receive this option and adherence to medications is often low. This 
+intervention will assess the impact of a polypill on SBP and CVD. 
 
-**Source information:**
+Please note that as the polypill only includes hypertension medications, LDL-C adherence will
```

Comment:
This confuses me. Are you saying that this pill is to be treated only as a blood pressure pill and so SBP adherence is to be used when determining whether a person takes it?

Reply:
Yes, but I'll rephrase in the docs too!